src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -308,7 +308,10 @@ private void OnExpanderButtonPreviewMouseMove(object sender, MouseEventArgs e)
                 return;
 
             var senderButton = e.OriginalSource as FrameworkElement;
-            dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null));
+            var searchElementVM = senderButton.DataContext as NodeSearchElementViewModel;
+            if (searchElementVM != null)
+                dragDropHelper.HandleMouseMove(senderButton, e.GetPosition(null)); 
+
```

Comment:
And here is right place for fix. And you've even made this fix! :) Problem is that sender can be RootSearchElementVM or ClassInformationViewModel. And we should just fire `HandleMouseMove`, when ViewModel is `NodeSearchElementViewModel`. In other cases we don't need handle mouse move.